pysite/views/staff/tables/edit.py

```diff
@@ -0,0 +1,112 @@
+import json
+
+from flask import redirect, request, url_for
+from werkzeug.exceptions import BadRequest, NotFound
+
+from pysite.base_route import RouteView
+from pysite.constants import TABLE_MANAGER_ROLES
+from pysite.decorators import csrf, require_roles
+from pysite.mixins import DBMixin
+from pysite.tables import TABLES
+
+DEFAULT_OLD_PRIMARY = "!None!"
```

Comment:
can you explain what this is all about? never seen anything like `!None!` before, almost looks spanish 😛. Why can't it just be `None`?